From 49af2bbed4eeab420b72781dbbb4b0a1fcac1ea9 Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Tue, 2 Feb 2021 14:10:02 +0200
Subject: [PATCH 59/89] iot-gate-imx8: dts: add CAN bus RX interrupt

The updated CAN bus driver supports the RX interrupt.
Add the definition for the RX interrupt GPIOs.

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/sb-iotgimx8-ied.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/compulab/sb-iotgimx8-ied.dts b/arch/arm64/boot/dts/compulab/sb-iotgimx8-ied.dts
index 23c267e5f4a9..430561babc15 100644
--- a/arch/arm64/boot/dts/compulab/sb-iotgimx8-ied.dts
+++ b/arch/arm64/boot/dts/compulab/sb-iotgimx8-ied.dts
@@ -92,6 +92,7 @@
 	can0: can@0 {
 		compatible = "microchip,mcp2518fd";
 		reg = <0>;
+		microchip,rx-int = <&gpio4 31 IRQ_TYPE_LEVEL_LOW>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_can0>;
 		interrupt-parent = <&gpio5>;
@@ -112,6 +113,7 @@
 	can1: can@0 {
 		compatible = "microchip,mcp2518fd";
 		reg = <0>;
+		microchip,rx-int = <&gpio5 28 IRQ_TYPE_LEVEL_LOW>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_can1>;
 		interrupt-parent = <&gpio5>;
@@ -203,12 +205,14 @@
 		pinctrl_can0: can0grp {
 			fsl,pins = <
 				MX8MM_IOMUXC_SAI3_TXC_GPIO5_IO0		0x00
+				MX8MM_IOMUXC_SAI3_TXFS_GPIO4_IO31       0x00
 			>;
 		};
 
 		pinctrl_can1: can1grp {
 			fsl,pins = <
 				MX8MM_IOMUXC_UART4_TXD_GPIO5_IO29	0x00
+				MX8MM_IOMUXC_UART4_RXD_GPIO5_IO28	0x00
 			>;
 		};
 	};
-- 
2.17.1

